### ISSUE TITLE
add filtering to url so that back button works as expected

### ISSUE THREAD

**Issue by trondhindenes** [piccolo-orm/piccolo_admin#163]
If I'm in a filtered list, and then click an item for details and then click back, I'm taken to the unfiltered view. It would be good if filters were expressed using url query params so that back navigation would take me back to the filtered view.

**Comment by sinisaos** [comment]
@trondhindenes There is already a [issue](https://github.com/piccolo-orm/piccolo_admin/issues/42) with that problem, but progress has somehow stalled.

**Comment by dantownsend** [comment]
Yeah, I agree this needs adding. I've increased the priority of the issue.
